logr.go

```diff
@@ -314,6 +314,11 @@ func (l Logger) V(level int) Logger {
 	return l
 }
 
+// GetV returns the verbosity level of the logger.
```

Comment:
If we're going to keep the "all Discard loggers are identical" property, can we comment here like:

"""
GetV returns the verbosity level of the logger.  If the logger's LogSink is nil (see GetSink), this will always return 0.
"""

In fact maybe make it really clear:

```
func (l Logger) GetV() int {
    if l.sink == nil {
        return 0
    }
	return l.level
}
```
?